TestVectorGenerator/EvenMoreComplex.cs

```diff
@@ -100,6 +100,7 @@ public void Generate(Stream outputStream)
 			document.UsageRules.Add(new UsageRule
 			{
 				KeyId = document.ContentKeys.First().Id,
+				IntendedTrackType = "髬 髬髬",
```

Comment:
I don't think special characters in a specific field qualify as an "even more complex". Perhaps use readable characters here as well or omit values altogether.